class/defaults.yml

```diff
@@ -2,3 +2,12 @@ parameters:
   stackgres_operator:
     =_metadata: {}
     namespace: syn-stackgres-operator
+    charts:
+      stackgres-operator: 1.4.0
```

Comment:
Please use the following structure for chart dependencies: https://syn.tools/syn/explanations/commodore-components/helm-charts.html

That way our renovate will be able to update it